src/redis_lock/__init__.py

```diff
@@ -238,15 +239,21 @@ def extend(self, expire=None):
         elif error:
             raise RuntimeError("Unsupported error code %s from EXTEND script" % error)
 
-    def _lock_renewer(self, interval):
+    @staticmethod
+    def _lock_renewer(lockref, interval, stop):
         """
         Renew the lock key in redis every `interval` seconds for as long
         as `self._lock_renewal_thread.should_exit` is False.
         """
         log = getLogger("%s.lock_refresher" % __name__)
```

Comment:
I probably missed this in the previous changes, but I think `log` should be at module level (`lock_renewer_logger` or something).